Figure_02_02.py

```diff
@@ -1,11 +1,11 @@
 import matplotlib.pyplot as plt
 
-from utils import tf
-from utilsplot import bode
-
-G = tf([5], [10, 1], deadtime=2)
+from control import bode_plot, tf, pade
+from numpy import linspace
 
+G = tf([5], [10, 1])        #Define the transfer function
+num, den = pade(2, n=10)     #Generate a 10th order pade approximation for 2s delay
+P = tf(num, den)            #General transfer function of the delay
+Gdelay = G*P                #Apply the delay to the transfer function
```

Comment:
What is the benefit of replacing an exact version of the bode plot which handles dead time correctly with an approximation?